src/mirakuru/http.py

```diff
@@ -75,8 +83,19 @@ def after_start_check(self):
         """Check if defined URL returns expected status to a HEAD request."""
         try:
             conn = HTTPConnection(self.host, self.port)
-
-            conn.request('HEAD', self.url.path)
+            if self.payload and self.method in ('POST', 'PUT', 'PATCH'):
+                headers = {
+                    "Content-type": "application/x-www-form-urlencoded",
```

Comment:
If users can send a custom payload then maybe they should be able to specify the `Content-type` too?

Reply:
I'm barely familiar with HTTPConnection, but it seems no headers are needed here.

But good point. Custom headers might also bee needed.